src/Elastic.Apm/Report/Serialization/SerializationUtils.cs

```diff
@@ -0,0 +1,20 @@
+using Elastic.Apm.Helpers;
+
+namespace Elastic.Apm.Report.Serialization
+{
+	internal static class SerializationUtils
+	{
+		internal static string TrimToLength(string input, int maxLength)
+		{
+			input.ThrowIfArgumentNull(nameof(input));
+
+			if (input.Length <= maxLength) return input;
+
+			if (maxLength <= 5) return input.Substring(0, maxLength);
```

Comment:
Why is this a special case? We wanna avoid "..." if we end up with very short strings?

Reply:
I thought that it doesn't make much sense not leaving anything from the original string but I'm open to suggestions. Although it's only used by serialization and `Consts.PropertyMaxLength` is much higher than 5.

Reply:
Yeah, I think it's fine to leave it there. I agree that showing `"st..."` (for `string`)  would not be very helpful :)